html/components.html

```diff
@@ -0,0 +1,285 @@
+                  style="
+                    width: 506px;
+                    height: 200px;
+                    border: 0;
+                    transform: scale(1);
+                    overflow: hidden;
+                  "
+                  sandbox="allow-scripts allow-same-origin"
+                >
+                </iframe>
+              </div>
+            </div>
+            
+          </li>
+          <li id="navigation">
+            <div>
+              <h3>Navigation</h3>
+              <nav class="navbar">
+                <ul class="nav">
+                  <li class="nav-item">
+                    <a href="#!" class="nav-link">Link 1</a>
+                  </li>
+                  <li class="nav-item">
+                    <a href="#!" class="nav-link">Link 1</a>
+                  </li>
+                  <li class="nav-item">
+                    <a href="#!" class="nav-link">Link 1</a>
+                  </li>
+                  <li class="nav-item">
+                    <a href="#!" class="nav-link">Link 1</a>
+                  </li>
+                </ul>
+              </nav>
+
+              <div class="code-div">
+                <iframe
+                  src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=night-owl&wt=sharp&l=htmlmixed&ds=false&dsyoff=23px&dsblur=71px&wc=true&wa=true&pv=0px&ph=0px&ln=true&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cnav%2520class%253D%2522navbar%2522%253E%250A%2509%253Cul%2520class%253D%2522nav%2522%253E%250A%2520%2520%2520%2520%2509%253Cli%2520class%253D%2522nav-item%2522%253E%253Ca%2520href%253D%2522%2523%21%2522%2520class%253D%2522nav-link%2522%253ELink%25201%253C%252Fa%253E%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522nav-item%2522%253E%253Ca%2520href%253D%2522%2523%21%2522%2520class%253D%2522nav-link%2522%253ELink%25201%253C%252Fa%253E%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522nav-item%2522%253E%253Ca%2520href%253D%2522%2523%21%2522%2520class%253D%2522nav-link%2522%253ELink%25201%253C%252Fa%253E%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522nav-item%2522%253E%253Ca%2520href%253D%2522%2523%21%2522%2520class%253D%2522nav-link%2522%253ELink%25201%253C%252Fa%253E%253C%252Fli%253E%2520%250A%2509%253C%252Ful%253E%250A%253C%252Fnav%253E"
+                  style="
+                    width: 700px;
+                    height: 218px;
+                    border: 0;
+                    transform: scale(1);
+                    overflow: hidden;
+                  "
+                  sandbox="allow-scripts allow-same-origin"
+                >
+                </iframe>
+              </div>
+            </div>
+            
+          </li>
+          <li id="text">
```

Comment:
Same as above, you add space here or format the code with any vs code extensions. It will help you in neat and clean code.